creator-node/src/config.js

```diff
@@ -575,13 +581,37 @@ const config = convict({
     doc: 'Max number of sync failures for a secondary for a user per day before stopping further SyncRequest issuance',
     format: 'nat',
     env: 'secondaryUserSyncDailyFailureCountThreshold',
-    default: 10
+    default: 20
   },
   maxSyncMonitoringDurationInMs: {
     doc: 'Max duration that primary will monitor secondary for syncRequest completion',
     format: 'nat',
     env: 'maxSyncMonitoringDurationInMs',
     default: 300000 // 5min (prod default)
+  },
+  syncMaxCIDFailureCountBeforeSkip: {
+    doc: 'Max number of failures to retrieve CID before skipping over it during sync',
+    format: 'nat',
+    env: 'syncMaxCIDFailureCountBeforeSkip',
+    default: 10
+  },
+  skippedCIDsRetryQueueJobIntervalMs: {
+    doc: 'Interval (ms) for SkippedCIDsRetryQueue Job Processing',
+    format: 'nat',
+    env: 'skippedCIDsRetryQueueJobIntervalMs',
+    default: 3600000 // 1hr in ms
+  },
+  skippedCIDRetryQueueMaxAgeHr: {
```

Comment:
to prevent this job from infinitely growing. i set the maxAge pretty high, can make it even higher if you want
i was thinking actually having a second job with a much higher interval like once every 1-2 days that processes each CID

wdyt

Reply:
@dmanjunath bump

Reply:
resolved offline, will keep this way